### PR TITLE
[chore]Add Metric variance tests for multiclass and multilabel

### PR DESCRIPTION
**Pull request by wanjiku-carol**
#### What does this PR do?
Adds tests for metric variance tests for multiclass and multilabel
#### Description of Task to be completed?
open sklearn/metrics/tests/test_common.py, add the tests
#### How should this be manually tested?
Running sklearn -v on the cloned repo to see if the tests pass after addition
#### Any background context you want to provide?
[https://github.com/scikit-learn/scikit-learn/pull/8601](url)
#### What are the relevant pivotal tracker stories?
#146674771

### CODE REVIEW

File: sklearn/metrics/tests/test_common.py [hunk 1]
```diff
@@ -369,11 +369,11 @@
...
-                        # not work for confusion_matrix, as its output is a
-                        # matrix instead of a number. Testing of
-                        # confusion_matrix with sample_weight is in
-                        # test_classification.py
+    "confusion_matrix",  # Left this one here because the tests in this file do
+    # not work for confusion_matrix, as its output is a
+    # matrix instead of a number. Testing of
+    # confusion_matrix with sample_weight is in
+    # test_classification.py
     "median_absolute_error",
```
Comment (oluwayetty) [inline sklearn/metrics/tests/test_common.py]: Any reason why this had to be edited from how it was previously @wanjiku-carol ?

File: sklearn/metrics/tests/test_common.py [hunk 2]
```diff
@@ -619,9 +619,9 @@ def test_invariance_string_vs_numbers_labels():
...
 def test_inf_nan_input():
-    invalids =[([0, 1], [np.inf, np.inf]),
-               ([0, 1], [np.nan, np.nan]),
-               ([0, 1], [np.nan, np.inf])]
+    invalids = [([0, 1], [np.inf, np.inf]),
+                ([0, 1], [np.nan, np.nan]),
+                ([0, 1], [np.nan, np.inf])]
 
     METRICS = dict()
     METRICS.update(THRESHOLDED_METRICS)
```
Comment (oluwayetty) [inline sklearn/metrics/tests/test_common.py]: Please, kindly remove all the white spaces in this PR. An example is this.

File: sklearn/metrics/tests/test_common.py [hunk 3]
```diff
@@ -1110,3 +1110,101 @@ def test_no_averaging_labels():
...
+            y_true[i] = np.random.rand(n_classes) > p
+        y_score.append(y_true[i] + rng.normal(size=y_true[i].shape))
+        y_score_perm.append(y_score[i][classes_perm])
+
+    y_true_perm = y_true[:, classes_perm]
+    for name in THRESHOLDED_MULTILABEL_METRICS:
+        metric = ALL_METRICS[name]
+        score = metric(y_true, y_score)
+        score_perm = metric(y_true_perm, y_score_perm)
+        assert_almost_equal(score, score_perm)
```
Comment (oluwayetty) [inline sklearn/metrics/tests/test_common.py:1210]: Nice work here ladies, you all seem to have really catch up fast 👍 
The PR will be good to go when the other comments have been re-accessed. Thanks

**Review by oluwayetty — CHANGES_REQUESTED** [review]

File: sklearn/metrics/tests/test_common.py (hunk 1 above)
Comment (wanjiku-carol) [reply sklearn/metrics/tests/test_common.py]: This was as a result of the auto-pep8 in my atom text editor that corrected the spacing automatically. I have returned it as it was.

File: sklearn/metrics/tests/test_common.py (hunk 2 above)
Comment (wanjiku-carol) [reply sklearn/metrics/tests/test_common.py]: This was also a result of the auto-pep8 in my atom text editor that corrected the spacing automatically. I have returned it as it was.
